src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -163,6 +167,33 @@ public String toString() {
         return groups.asUnmodifiableObservableMap();
     }
 
+    @Override
+    public ObservableList<PersonEvent> getUpcomingDates() {
+        List<PersonEvent> personEvents = new ArrayList<>();
+        persons.forEach(person -> {
+            // Add birthday
+            personEvents.add(new PersonEvent(person.getBirthday().getBirthday(), person,
```

Comment:
`person.getBirthday().getBirthday()` is very strange >_<. Can rename to the last `getBirthday()` method to `getDate()`?